localized/ja/01-Navigation/4-Contextual_navigation/4.2-Navigate_To_menu_on_type.cs

```diff
@@ -46,7 +46,7 @@ public class DerivedClass   // <- DerivedClass を選択してキャレットを
     //
     // (参考) 候補間の移動方法
     //
-    // TODO: Rider のショートカットはデフォルトで未定義(設定なし)であり、独自に設定しなければ動作しない
+    // NOTE: Rider のショートカットはデフォルトで未定義(設定なし)であり、独自に設定しなければ動作しない
     // [Settings] > [Keymap] > [Main Manu] > [Edit] > [Find Usages]
```

Comment:
Was thinking it may make sense to also use `|` here to separate menu's. However, if you think this is better with `[ ] > ....` in the localized version that works. What is the typical approach you see in other Japanese documentation?

Reply:
Hi, @maartenba .

I haven't seen much of the " menu-lv1 | menu-lv2 | ..." format in Japanese documentation.
At least for me, the "[menu-lv1] > [menu-lv2] > ..." format is more intuitive and easier to understand.

I think it would be better to keep the "[menu-lv1] > [menu-lv2] > ..." in the Japanese localized version.